docs/src/pages/WebCheckboxPage/WebCheckboxPage.js

```diff
@@ -141,6 +141,31 @@ const components = [
       </form>,
     ],
   },
+  {
+    id: 'light',
+    title: 'Light',
```

Comment:
Should we call this `white` to match the prop name?

```suggestion
    title: 'White',
```

Reply:
I was thinking that - but also thinking about inclusive language 🤔

Reply:
Ahh, yeah. I think we're literally describing a colour though so it's ok — it's not like whitelist vs allowlist